### DISCUSSION TITLE
剪贴板历史支持跳过密码吗

### DISCUSSION THREAD

**Discussion by duament** [fcitx/fcitx5#1002] [Q&A] [answered]
之前用KDE的时候，自带的剪贴板历史不会记录密码，现在换hyprland之后，用fcitx5的剪贴板会保存密码，看了一下keepassxc复制出来的密码有一个`x-kde-passwordManagerHint`，fcitx5支持这个吗

```
❯ wl-paste -l
text/plain
x-kde-passwordManagerHint
text/plain;charset=utf-8
```

**Comment by wengxt** [comment, marked as answer]
头一次听说有这种功能，可以支持一下

**Reply by wengxt** [reply]
5.1.9 中支持，不过默认还是进入的，但是定时清除

![image](https://github.com/fcitx/fcitx5/assets/259684/9edc161b-201d-4573-ac1c-b05630ba5d63)
